src/game/client.cpp

```diff
@@ -477,11 +477,14 @@ namespace client
                 items.emplace_back(v >= W_OFFSET && v < W_ITEM ? v : 0);
             }
         }
-        game::player1.loadweap.shrink(0);
-        loopv(items) if(game::player1.loadweap.find(items[i]) < 0)
-        {
-            game::player1.loadweap.add(items[i]);
-            if(game::player1.loadweap.length() >= W_LOADOUT) break;
+        game::player1.loadweap.clear();
+        for (const auto &i : items) {
+            if (std::find(game::player1.loadweap.begin(),
```

Comment:
Why do you have to use `std::end` over `loadweap.end()`?

Reply:
No idea what I thought of when writing that...

I'm also guessing it is better to use `std::any_of`?

Reply:
It saves you a little bit of code, and also helps make clear what you're doing here. I guess you want to use `std::none_of`, though.